packages/app/src/templates/components/template-options/template-options.component.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { Actions } from "App/__deprecated__/renderer/components/core/table/table.component"
+import Dropdown from "App/__deprecated__/renderer/components/core/dropdown/dropdown.component"
+import { IconButtonWithSecondaryTooltip } from "App/__deprecated__/renderer/components/core/icon-button-with-tooltip/icon-button-with-secondary-tooltip.component"
+import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"
+import { ElementWithTooltipPlace } from "App/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component"
+import ButtonComponent from "App/__deprecated__/renderer/components/core/button/button.component"
+import { DisplayStyle } from "App/__deprecated__/renderer/components/core/button/button.config"
+import { TemplateOptionsProps } from "App/templates/components/template-options/template-options.interface"
+import { TemplateOptionsTestIds } from "App/templates/components/template-options/template-options-test-ids.enum"
+
+const messages = defineMessages({
+  dropdownTogglerTooltipDescription: {
+    id: "component.dropdownTogglerTooltipDescription",
```

Comment:
Is it the correct id? I couldn't find it in `en-US.json`.

![image](https://user-images.githubusercontent.com/41268731/173572743-3f363266-91d5-4514-8bf4-3ff656ca4bc7.png)

Reply:
https://github.com/mudita/mudita-center/pull/781/commits/7498e616c570ec4a60e65142608e52b9011156a1